src/components/Sum.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({
+  a,
+  b,
+}) => (
+
+  <p className="MessageSum">
+    Sum of
+    &nbsp;
+    {a}
+    &nbsp;
+    and
+    &nbsp;
+    {b}
+    &nbsp;
+    is
+    &nbsp;
+    {a + b}
```

Comment:
```suggestion
    {`Sum of ${a} and ${b} is ${a + b}`}
```
Лучше использовать синтаксис из js